src/proof/PGCheck.cc

```diff
@@ -21,102 +21,55 @@ along with Periplo. If not, see <http://www.gnu.org/licenses/>.
 #include "PG.h"
 
 #include "OsmtInternalException.h"
+#include "VerificationUtils.h"
 
 #include <deque>
-#include <sys/wait.h>
-#include <unistd.h>
 
 
 void ProofGraph::verifyLeavesInconsistency( )
 {
-	if( verbose() > 0 ) { std::cerr << "# Verifying unsatisfiability of the set of proof leaves" << std::endl; }
+    if (verbose() > 0) { std::cerr << "# Verifying unsatisfiability of the set of proof leaves" << std::endl; }
 
 	std::vector<clauseid_t> proofleaves;
-	clauseid_t id,id1,id2;
 	std::vector<clauseid_t> q;
-	ProofNode * node = NULL;
-	std::vector<unsigned>* visited_count_ = new std::vector<unsigned>(getGraphSize(),0);
-	std::vector<unsigned>& visited_count = *visited_count_;
-	// Building DFS vector
+	std::vector<unsigned> visited_count(getGraphSize(), 0u);
 	q.push_back(getRoot()->getId());
```

Comment:
There seems to be an indentation problem.

Reply:
Fixed.